src/intro-tailwindcss.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
```

Comment:
Thoughts on structuring this in subfolders so it'd be something like mangotree.dev/resources/intro-tailwindcss or mangotree.dev/articles/intro-tailwindcss?

Reply:
Perfect, just structured it so it's in an articles folder and the url will be something like `mangotree.dev/blog/intro-tailwindcss` so if follows up with the `mangotree.dev/blog`